2025-07-08 17:16:26 The virtual environment found in /app/.venv seems to be broken.
2025-07-08 17:16:26 Recreating virtualenv spend-time-together in /app/.venv
2025-07-08 17:16:35 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:35 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:35 INFO  [alembic.runtime.migration] Running upgrade  -> 2fcfad900da8, Create users table
2025-07-08 17:16:38 Traceback (most recent call last):
2025-07-08 17:16:38   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:38     sys.exit(main())
2025-07-08 17:16:38              ^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:38     return self.main(*args, **kwargs)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:38     rv = self.invoke(ctx)
2025-07-08 17:16:38          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:38     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:38     return callback(*args, **kwargs)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:38     run(
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:38     server.run()
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:38     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:38     return runner.run(main)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:38     return self._loop.run_until_complete(task)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:38     return future.result()
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:38     await self._serve(sockets)
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:38     config.load()
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:38     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:38                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:38     module = importlib.import_module(module_str)
2025-07-08 17:16:38              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:38     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:38            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:38   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:38   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:38   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:38   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:38   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:38   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:38     from app.api.routes import api_router
2025-07-08 17:16:38   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:38     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:38   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:38     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:38   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:38     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:38   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:38     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:38                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:38   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:38     raise AttributeError(item)
2025-07-08 17:16:38 AttributeError: Field
2025-07-08 17:16:42 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:42 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:43 Traceback (most recent call last):
2025-07-08 17:16:43   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:43     sys.exit(main())
2025-07-08 17:16:43              ^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:43     return self.main(*args, **kwargs)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:43     rv = self.invoke(ctx)
2025-07-08 17:16:43          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:43     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:43     return callback(*args, **kwargs)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:43     run(
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:29 The --no-wheel and --wheel options are deprecated. They have no effect for Python > 3.8 as wheel is no longer bundled in virtualenv.
2025-07-08 17:16:43     server.run()
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:43     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:43     return runner.run(main)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:43     return self._loop.run_until_complete(task)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:43     return future.result()
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:43     await self._serve(sockets)
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:43     config.load()
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:43     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:43                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:43     module = importlib.import_module(module_str)
2025-07-08 17:16:43              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:43     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:43            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:43   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:43   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:43   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:43   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:43   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:43   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:43     from app.api.routes import api_router
2025-07-08 17:16:43   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:43     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:43   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:43     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:43   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:43     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:43   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:43     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:43                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:43   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:43     raise AttributeError(item)
2025-07-08 17:16:43 AttributeError: Field
2025-07-08 17:16:46 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:46 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:48 Traceback (most recent call last):
2025-07-08 17:16:48   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:48     sys.exit(main())
2025-07-08 17:16:48              ^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:48     return self.main(*args, **kwargs)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:48     rv = self.invoke(ctx)
2025-07-08 17:16:48          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:48     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:48     return callback(*args, **kwargs)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:48     run(
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:48     server.run()
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:48     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:48     return runner.run(main)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:48     return self._loop.run_until_complete(task)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:48     return future.result()
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:48     await self._serve(sockets)
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:48     config.load()
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:48     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:48                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:48     module = importlib.import_module(module_str)
2025-07-08 17:16:48              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:48     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:48   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:48   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:48   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:48   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:48   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:48   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:48     from app.api.routes import api_router
2025-07-08 17:16:48   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:48     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:48   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:48     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:48   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:48     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:48   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:48     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:48                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:48   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:48     raise AttributeError(item)
2025-07-08 17:16:48 AttributeError: Field
2025-07-08 17:16:50 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:50 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:51 Traceback (most recent call last):
2025-07-08 17:16:51   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:51     sys.exit(main())
2025-07-08 17:16:51              ^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:51     return self.main(*args, **kwargs)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:51     rv = self.invoke(ctx)
2025-07-08 17:16:51          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:51     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:51     return callback(*args, **kwargs)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:51     run(
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:51     server.run()
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:51     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:51     return runner.run(main)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:51     return self._loop.run_until_complete(task)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:51     return future.result()
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:51     await self._serve(sockets)
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:51     config.load()
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:51     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:51                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:51     module = importlib.import_module(module_str)
2025-07-08 17:16:51              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:51     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:51            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:51   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:51   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:51   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:51   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:51   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:51   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:51     from app.api.routes import api_router
2025-07-08 17:16:51   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:51     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:51   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:51     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:51   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:51     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:51   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:51     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:51                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:51   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:51     raise AttributeError(item)
2025-07-08 17:16:51 AttributeError: Field
2025-07-08 17:16:54 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:54 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:55 Traceback (most recent call last):
2025-07-08 17:16:55   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:55     sys.exit(main())
2025-07-08 17:16:55              ^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:55     return self.main(*args, **kwargs)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:55     rv = self.invoke(ctx)
2025-07-08 17:16:55          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:55     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:55     return callback(*args, **kwargs)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:55     run(
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:55     server.run()
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:55     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:55     return runner.run(main)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:55     return self._loop.run_until_complete(task)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:55     return future.result()
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:55     await self._serve(sockets)
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:55     config.load()
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:55     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:55                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:55     module = importlib.import_module(module_str)
2025-07-08 17:16:55              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:55     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:55            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:55   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:55   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:55   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:55   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:55   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:55   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:55     from app.api.routes import api_router
2025-07-08 17:16:55   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:55     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:55   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:55     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:55   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:55     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:55   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:55     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:55                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:55   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:55     raise AttributeError(item)
2025-07-08 17:16:55 AttributeError: Field
2025-07-08 17:16:58 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:16:58 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:16:58 Traceback (most recent call last):
2025-07-08 17:16:58   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:16:58     sys.exit(main())
2025-07-08 17:16:58              ^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:16:58     return self.main(*args, **kwargs)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:16:58     rv = self.invoke(ctx)
2025-07-08 17:16:58          ^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:16:58     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:16:58     return callback(*args, **kwargs)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:16:58     run(
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:16:58     server.run()
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:16:58     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:16:58     return runner.run(main)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:16:58     return self._loop.run_until_complete(task)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:16:58     return future.result()
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:16:58     await self._serve(sockets)
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:16:58     config.load()
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:16:58     self.loaded_app = import_from_string(self.app)
2025-07-08 17:16:58                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:16:58     module = importlib.import_module(module_str)
2025-07-08 17:16:58              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:16:58     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:16:58            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:16:58   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:16:58   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:16:58   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:16:58   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:16:58   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:16:58   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:16:58     from app.api.routes import api_router
2025-07-08 17:16:58   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:16:58     from app.api.auth.controller import router as auth_router
2025-07-08 17:16:58   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:16:58     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:16:58   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:16:58     class AuthUserSerializer(BaseModel):
2025-07-08 17:16:58   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:16:58     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:16:58                  ^^^^^^^^^^^^^^^
2025-07-08 17:16:58   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:16:58     raise AttributeError(item)
2025-07-08 17:16:58 AttributeError: Field
2025-07-08 17:17:02 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:17:02 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:17:03 Traceback (most recent call last):
2025-07-08 17:17:03   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:17:03     sys.exit(main())
2025-07-08 17:17:03              ^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:17:03     return self.main(*args, **kwargs)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:17:03     rv = self.invoke(ctx)
2025-07-08 17:17:03          ^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:17:03     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:17:03     return callback(*args, **kwargs)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:17:03     run(
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:17:03     server.run()
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:17:03     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:17:03     return runner.run(main)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:17:03     return self._loop.run_until_complete(task)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:17:03     return future.result()
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:17:03     await self._serve(sockets)
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:17:03     config.load()
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:17:03     self.loaded_app = import_from_string(self.app)
2025-07-08 17:17:03                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:17:03     module = importlib.import_module(module_str)
2025-07-08 17:17:03              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:17:03     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:17:03            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:17:03   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:17:03   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:17:03   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:17:03   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:17:03   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:17:03   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:17:03     from app.api.routes import api_router
2025-07-08 17:17:03   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:17:03     from app.api.auth.controller import router as auth_router
2025-07-08 17:17:03   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:17:03     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:17:03   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:17:03     class AuthUserSerializer(BaseModel):
2025-07-08 17:17:03   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:17:03     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:17:03                  ^^^^^^^^^^^^^^^
2025-07-08 17:17:03   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:17:03     raise AttributeError(item)
2025-07-08 17:17:03 AttributeError: Field
2025-07-08 17:17:10 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:17:10 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:17:11 Traceback (most recent call last):
2025-07-08 17:17:11   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:17:11     sys.exit(main())
2025-07-08 17:17:11              ^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:17:11     return self.main(*args, **kwargs)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:17:11     rv = self.invoke(ctx)
2025-07-08 17:17:11          ^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:17:11     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:17:11     return callback(*args, **kwargs)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:17:11     run(
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:17:11     server.run()
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:17:11     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:17:11     return runner.run(main)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:17:11     return self._loop.run_until_complete(task)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:17:11     return future.result()
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:17:11     await self._serve(sockets)
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:17:11     config.load()
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:17:11     self.loaded_app = import_from_string(self.app)
2025-07-08 17:17:11                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:17:11     module = importlib.import_module(module_str)
2025-07-08 17:17:11              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:17:11     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:17:11            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:17:11   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:17:11   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:17:11   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:17:11   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:17:11   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:17:11   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:17:11     from app.api.routes import api_router
2025-07-08 17:17:11   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:17:11     from app.api.auth.controller import router as auth_router
2025-07-08 17:17:11   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:17:11     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:17:11   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:17:11     class AuthUserSerializer(BaseModel):
2025-07-08 17:17:11   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:17:11     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:17:11                  ^^^^^^^^^^^^^^^
2025-07-08 17:17:11   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:17:11     raise AttributeError(item)
2025-07-08 17:17:11 AttributeError: Field
2025-07-08 17:17:25 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:17:25 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:17:26 Traceback (most recent call last):
2025-07-08 17:17:26   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:17:26     sys.exit(main())
2025-07-08 17:17:26              ^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:17:26     return self.main(*args, **kwargs)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:17:26     rv = self.invoke(ctx)
2025-07-08 17:17:26          ^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:17:26     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:17:26     return callback(*args, **kwargs)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:17:26     run(
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:17:26     server.run()
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:17:26     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:17:26     return runner.run(main)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:17:26     return self._loop.run_until_complete(task)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:17:26     return future.result()
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:17:26     await self._serve(sockets)
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:17:26     config.load()
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:17:26     self.loaded_app = import_from_string(self.app)
2025-07-08 17:17:26                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:17:26     module = importlib.import_module(module_str)
2025-07-08 17:17:26              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:17:26     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:17:26            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:17:26   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:17:26   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:17:26   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:17:26   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:17:26   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:17:26   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:17:26     from app.api.routes import api_router
2025-07-08 17:17:26   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:17:26     from app.api.auth.controller import router as auth_router
2025-07-08 17:17:26   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:17:26     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:17:26   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:17:26     class AuthUserSerializer(BaseModel):
2025-07-08 17:17:26   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:17:26     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:17:26                  ^^^^^^^^^^^^^^^
2025-07-08 17:17:26   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:17:26     raise AttributeError(item)
2025-07-08 17:17:26 AttributeError: Field
2025-07-08 17:17:53 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:17:53 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:17:53 Traceback (most recent call last):
2025-07-08 17:17:53   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:17:53     sys.exit(main())
2025-07-08 17:17:53              ^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:17:53     return self.main(*args, **kwargs)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:17:53     rv = self.invoke(ctx)
2025-07-08 17:17:53          ^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:17:53     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:17:53     return callback(*args, **kwargs)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:17:53     run(
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:17:53     server.run()
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:17:53     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:17:53     return runner.run(main)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:17:53     return self._loop.run_until_complete(task)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:17:53     return future.result()
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:17:53     await self._serve(sockets)
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:17:53     config.load()
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:17:53     self.loaded_app = import_from_string(self.app)
2025-07-08 17:17:53                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:17:53     module = importlib.import_module(module_str)
2025-07-08 17:17:53              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:17:53     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:17:53            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:17:53   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:17:53   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:17:53   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:17:53   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:17:53   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:17:53   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:17:53     from app.api.routes import api_router
2025-07-08 17:17:53   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:17:53     from app.api.auth.controller import router as auth_router
2025-07-08 17:17:53   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:17:53     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:17:53   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:17:53     class AuthUserSerializer(BaseModel):
2025-07-08 17:17:53   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:17:53     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:17:53                  ^^^^^^^^^^^^^^^
2025-07-08 17:17:53   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:17:53     raise AttributeError(item)
2025-07-08 17:17:53 AttributeError: Field
2025-07-08 17:18:46 INFO  [alembic.runtime.migration] Context impl MySQLImpl.
2025-07-08 17:18:46 INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025-07-08 17:18:47 Traceback (most recent call last):
2025-07-08 17:18:47   File "/usr/local/bin/uvicorn", line 8, in <module>
2025-07-08 17:18:47     sys.exit(main())
2025-07-08 17:18:47              ^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1442, in __call__
2025-07-08 17:18:47     return self.main(*args, **kwargs)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1363, in main
2025-07-08 17:18:47     rv = self.invoke(ctx)
2025-07-08 17:18:47          ^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1226, in invoke
2025-07-08 17:18:47     return ctx.invoke(self.callback, **ctx.params)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 794, in invoke
2025-07-08 17:18:47     return callback(*args, **kwargs)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 413, in main
2025-07-08 17:18:47     run(
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 580, in run
2025-07-08 17:18:47     server.run()
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in run
2025-07-08 17:18:47     return asyncio.run(self.serve(sockets=sockets))
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2025-07-08 17:18:47     return runner.run(main)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2025-07-08 17:18:47     return self._loop.run_until_complete(task)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
2025-07-08 17:18:47     return future.result()
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 71, in serve
2025-07-08 17:18:47     await self._serve(sockets)
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 78, in _serve
2025-07-08 17:18:47     config.load()
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 436, in load
2025-07-08 17:18:47     self.loaded_app = import_from_string(self.app)
2025-07-08 17:18:47                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
2025-07-08 17:18:47     module = importlib.import_module(module_str)
2025-07-08 17:18:47              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2025-07-08 17:18:47     return _bootstrap._gcd_import(name[level:], package, level)
2025-07-08 17:18:47            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2025-07-08 17:18:47   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2025-07-08 17:18:47   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2025-07-08 17:18:47   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2025-07-08 17:18:47   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2025-07-08 17:18:47   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2025-07-08 17:18:47   File "/app/app/main.py", line 7, in <module>
2025-07-08 17:18:47     from app.api.routes import api_router
2025-07-08 17:18:47   File "/app/app/api/routes.py", line 6, in <module>
2025-07-08 17:18:47     from app.api.auth.controller import router as auth_router
2025-07-08 17:18:47   File "/app/app/api/auth/controller.py", line 6, in <module>
2025-07-08 17:18:47     from app.api.auth.serializers import AuthUserSerializer, AuthUserResponseSerializer
2025-07-08 17:18:47   File "/app/app/api/auth/serializers.py", line 4, in <module>
2025-07-08 17:18:47     class AuthUserSerializer(BaseModel):
2025-07-08 17:18:47   File "/app/app/api/auth/serializers.py", line 5, in AuthUserSerializer
2025-07-08 17:18:47     login: str = BaseModel.Field(..., title="Логин пользователя", description="Уникальный логин для входа в систему")
2025-07-08 17:18:47                  ^^^^^^^^^^^^^^^
2025-07-08 17:18:47   File "/usr/local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 271, in __getattr__
2025-07-08 17:18:47     raise AttributeError(item)
2025-07-08 17:18:47 AttributeError: Field
